rpmconf/Env/crontabs/crontab.logger

```diff
@@ -12,3 +12,7 @@
 # Daily reports
 #
 30 23 * * * /opt/zimbra/libexec/zmdailyreport -m
+#
+# Password expiry reminders
+#
+0 0 * * * /opt/zimbra/bin/zmpasswordexpiryreminder
```

Comment:
Could you clarify why this was specifically added to `crontab.logger`?

Reply:
I think we can move this to crontab.store

Reply:
Moving to crontab.store will execute this script for all mailbox nodes, it will need to handle scenarios where accounts related to specific nodes only for those the script executes and sends email for the accounts for password expiry